src/vaadin-contextmenu-items-mixin.html

```diff
@@ -203,10 +203,10 @@
 
         if (component.localName === 'vaadin-context-menu-item') {
```

Comment:
This should probably be updated to:
```js
if (component instanceof Vaadin.ItemElement) {
```
to also work with other elements extending vaadin item.

Reply:
Done.